src/ui/windows/TogglDesktop/TogglDesktop/ui/windows/AboutWindow.xaml

```diff
@@ -25,7 +26,7 @@
     <i:Interaction.Behaviors>
         <behaviors:HideWindowOnClosingBehavior />

```

Comment:
Why hiding window instead of just closing it?

Reply:
It's just the way it had been previously implemented. Won't speculate on why, but I can say that with our app being rather small keeping those extra views in memory doesn't hurt our performance that much. To make it possible to close child windows instead of hiding we need to take care of even subscription/unsubscription, state, and the logic which is being executed even when the child windows are hidden.
This behavior class was introduced by refactoring the existing procedural code-behind logic into more declarative code. This also makes it easier to get rid of "hiding instead of closing" in the future.

Reply:
Closing window needs to take care of "Finalizing" on closing but it makes code cleaner and predictable, and in the opposite side hiding window needs to "resetting state" which done here by re-creating new  viewmodel and rebinding controls and maybe forget to reset some objects or controls could cause unpredictable behavior, this approach also makes the code harder to read.

I understand that its just the way it had been previously implemented, but I can't see any benefit of using it in this window specifically.